### ISSUE TITLE
Apply color theme

### ISSUE THREAD

**Issue by ryum91** [nhn/tui.editor#907]
## Version
latest

## Development Environment
none

## Current Behavior
Only white theme

## Expected Behavior
Apply color theme (ex. Dark mode, Custom theme)

**Comment by seonim-ryu** [comment]
@ryum91 Good idea! But I think it will be difficult right now. There are no APIs related to themes and I need to organize the specifications. Anyway, thanks for your comment. :)

**Comment by Dunky13** [comment]
It would be much preferred, or even apply custom themes?

**Comment by beeman** [comment]
One possible solution that might be relatively easy to implement is moving the colors to CSS variables, they seem [pretty well](https://caniuse.com/#feat=css-variables) supported.

That way the dev using the library can set their own values and override the editor's theme.	

I'm happy to take a stab at implementing that :)
